.codecov.yml

```diff
@@ -0,0 +1,5 @@
+fixes:
+  - "^.*/site-packages/opentelemetry/sdk/::opentelemetry-sdk/src/opentelemetry/sdk/"
```

Comment:
We may want to stop testing against site-packages to avoid this ugly hack and others like it.

Reply:
It's done.